core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/ProfileClassWarehouse.scala

```diff
@@ -136,22 +136,25 @@ case class RapidsJarProfileResult(appIndex: Int, jar: String)  extends ProfileRe
 
 case class DataSourceProfileResult(appIndex: Int, sqlID: Long, nodeId: Long,
     format: String, buffer_time: Long, scan_time: Long, data_size: Long,
-    decode_time: Long, location: String, pushedFilters: String, schema: String)
+    decode_time: Long, location: String, pushedFilters: String, schema: String,
+    dataFilters: String, partitionFilters: String)
 extends ProfileResult {
   override val outputHeaders =
     Seq("appIndex", "sqlID", "nodeId", "format", "buffer_time", "scan_time", "data_size",
-      "decode_time", "location", "pushedFilters", "schema")
+      "decode_time", "location", "pushedFilters", "schema", "data_filters", "partition_filters")
```

Comment:
Nit: Can we keep all the `filter` names consistent. Can make pushedFilters to pushed_filters.

Reply:
I did not want to change `pushedFilters` because it would break anyone reading that column.

Reply:
All columns are snakecase except the pushedFilters.